harness/features/initialize.local.test.ts

```diff
@@ -0,0 +1,153 @@
+import {
+    getConnectionStringForProxy,
+    forEachSDK,
+    createClient,
+    callOnClientInitialized,
+    describeIf,
+    wait
+} from '../helpers'
+import { v4 as uuidv4 } from 'uuid'
+import { getServerScope } from '../mockServer'
+import { Capabilities, SDKCapabilities } from '../types'
+import nock from 'nock'
+
+jest.setTimeout(10000)
+
+describe('Initialize Tests - Local', () => {
+    const mockServerUrl 
+        = `http://${process.env.DOCKER_HOST_IP ?? 'host.docker.internal'}:${global.__MOCK_SERVER_PORT__}`
+    const scope = getServerScope()
+
+    forEachSDK((name: string) => {
+        const capabilities: string[] = SDKCapabilities[name]
+        const sdkKey = 'server_SDK_KEY'
+        let url: string
+
+        beforeAll(async () => {
+            url = getConnectionStringForProxy(name)
+        })
+
+        afterEach(() => {
+            nock.cleanAll()
+        })
+
+        describeIf(capabilities.includes(Capabilities.local))(name, () => {
+            it('should error when SDK key is missing', async () => {
+                const clientId = uuidv4()
+                scope
+                    .get(`/client/${clientId}/config/v1/server/${sdkKey}.json`)
+                    .reply(200, {})
+
+                const response = await createClient(url, clientId)
```

Comment:
you can likely just create one client for this test block if you don't specifically need it to be a different client, jest runs all tests in a test file sequentially

Reply:
though since these are testing the initialize call I think it makes sense to create a separate client for each